storage/database/src/env.rs

```diff
@@ -81,13 +80,13 @@ pub trait Env: Sized {
     /// Open the database environment, using the passed [`Config`].
     ///
     /// # Invariants
-    /// This function **must** create all tables listed in [`crate::tables`].
+    /// This function does not create any tables.
```

Comment:
Another thing about `cuprate_database` not knowing about the user's tables:

There's going to have to be a "change table sort order" API created in `cuprate_database` for https://github.com/Cuprate/cuprate/issues/179.

Probably using `trait cuprate_database::Key`? Discussion for another time though.

Reply:
BTW this should have been done in the initial implementation, sorry.

Reply:
I think a good way to do it would be with an enum like:

```rust
enum Sort {
    Default,
    Integer,
    Custom(compare_func)
}
```

As some DBs (for example LMDB) have specific options for integer sorting which should be faster than doing custom sorting